### ISSUE TITLE
Can't set keyword `path` in data

### ISSUE THREAD

**Issue by M0NsTeRRR** [ansible-collections/community.hashi_vault#389]
##### SUMMARY

We can't set `path` key in `data`, for example to configure ACME path in PKI plugin.
The issue is in hvac library see https://github.com/hvac/hvac/issues/133
I've opened this issue to track it and add a fix when hvac will fix the issue

##### ISSUE TYPE
- Bug Report

##### ANSIBLE VERSION

```paste below
2.15.0
```

##### COLLECTION VERSION

```paste below
5.0.0
```

##### STEPS TO REPRODUCE

```yaml
---
- name: Create vault secret
  community.hashi_vault.vault_write:
    url: https://127.0.0.1:8200
    token: token
    path: pki/config/cluster
    data:
      path: https://vault-active.vault.svc:8200/v1/pki
```

##### EXPECTED RESULTS

Update `path` value of `pki/config/cluster`

##### ACTUAL RESULTS

```paste below
hvac.v1.Client.write() got multiple values for keyword argument 'path'
```

**Comment by M0NsTeRRR** [comment]
@briantist replying to your message here.

We can base the new code on `hvac.__version__`

**Comment by M0NsTeRRR** [comment]
Maybe something like this is better, I think we don't need to warn people that don't use 'path' or 'wrap_ttl' ?
```python
try:
    client.write_data(data)
except (NotImplementedError, AttributeError):
    if "path" in data or "wrap_ttl" in data:
        raise_from(
            AnsibleError("To support 'path' and 'wrap_ttl' key in 'data' parameter you must have hvac >= 1.2"),
            HashiVaultValueError
        )
    else:
        client.write(data)
```

**Comment by briantist** [comment]
That seems reasonable, yeah!

**Comment by briantist** [comment]
Fix released!
- https://github.com/ansible-collections/community.hashi_vault/releases/tag/5.0.1
- https://galaxy.ansible.com/ui/repo/published/community/hashi_vault/?version=5.0.1
